__tests__/modules/type-is.test.ts

```diff
@@ -0,0 +1,51 @@
+import { typeIs } from '../../packages/type-is/src'
+
+describe('typeIs', () => {
+  it('should return false when value is falsy', () => {
+    expect(typeIs('')).toBe(false)
+    expect(typeIs(null)).toBe(false)
+    expect(typeIs(undefined)).toBe(false)
+  })
+
+  it('should return value if types are empty', () => {
+    expect(typeIs('application/json')).toBe('application/json')
+  })
+
+  it("shouldn't depends on case", () => {
```

Comment:
the proper structure of the sentence is "shouldn't depend on case"

Reply:
oh, my English... 🤦‍♂️
done )